### PR TITLE
rewrite introduction

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

File: draft-lcurley-moq-transport.md [hunk 1]
```diff
@@ -60,9 +60,22 @@ Objects are starved/dropped during congestion based on priorities in order to mi
 
 
 ## Introduction
-MoQTransport is a live media transport protocol that utilizes the QUIC network protocol {{QUIC}},
-either directly or via WebTransport {{WebTransport}}.
-It was originally developed for live media, but has been generalized for similar use-cases.
+
+MoQTransport (MoQT) is a transport protocol that utilizes the QUIC
```
Comment (suhasHere) [inline draft-lcurley-moq-transport.md:64]: s/MoQT/moqt/g

File: draft-lcurley-moq-transport.md [hunk 2]
```diff
@@ -60,9 +60,22 @@ Objects are starved/dropped during congestion based on priorities in order to mi
...
+network protocol {{QUIC}}, either directly or via WebTransport
+{{WebTransport}}, for the dissemination of media. MoQT utilizes a
+publish/subscribe workflow in which producers of media publish data in
+response to subscription requests from a multiplicity of endpoints. MoQT
+supports live, as well as near-live and Video on Demand (VOD)
```
Comment (suhasHere) [inline draft-lcurley-moq-transport.md:69]: live, interactive, as well as near-live ....

**Review by suhasHere — APPROVED** [review]
Looks fine . It would be nice if editorial nits are addressed as you see appropriate

**Review by kixelated — CHANGES_REQUESTED** [review]

File: draft-lcurley-moq-transport.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (suhasHere) [reply draft-lcurley-moq-transport.md:69]: +1

[the remark this answers is not in the document]
Comment (suhasHere) [reply draft-lcurley-moq-transport.md:69]: I would suggest we edit it as

 MoQT
supports live, interactive use-cases.

I don;t understand near-live though

File: draft-lcurley-moq-transport.md [hunk 3]
```diff
@@ -60,9 +60,22 @@ Objects are starved/dropped during congestion based on priorities in order to mi
...
+use-cases. MoQT supports delivery over intermediate content distribution
+networks and is architected for high scale and low latency
+distribution. In live mode, MoQT facilitates a broad spectrum of latency
```
[the remark this answers is not in the document]
Comment (fluffy) [reply draft-lcurley-moq-transport.md:72]: I missed this, I agree we should pick some word other than "live mode"

**Comment by suhasHere** [comment]
@kixelated does the latest commit address your feedback? thanks

**Review by fluffy — COMMENTED** [review]
LGTM

File: draft-lcurley-moq-transport.md [hunk 4]
```diff
@@ -60,9 +60,22 @@ Objects are starved/dropped during congestion based on priorities in order to mi
...
+regimes, from real-time, to interactive and non-interactive.
+
+MoQTransport is a generic protocol is designed to work in concert with
+multiple MoQ Streaming Formats. These MoQ Streaming Formats define how
+content is encoded, packaged, and mapping to MoQT objects, along with
+policies for discovery, subscription and congestion response.
```
[the remark this answers is not in the document]
Comment (afrind) [reply draft-lcurley-moq-transport.md:74]: Streaming formats might specify strategies for setting priorities, or for producer congestion response - the sentence says "policies for..."  What's the right finessing here?

**Review by afrind — COMMENTED** [review]
lgtm modulo addressing Luke's comment.

**Review by kixelated — APPROVED** [review]

**Pull request merged**